src/sentry/search/events/constants.py

```diff
@@ -288,6 +288,29 @@ class ThresholdDict(TypedDict):
     "transaction.duration": "d:transactions/duration@millisecond",
     "user": "s:transactions/user@none",
 }
+# The assumed list of tags that all metrics have, some won't because we remove tags to reduce cardinality
+# Use the public api aliases here
+DEFAULT_METRIC_TAGS = {
```

Comment:
We just have to be a bit careful because instead of all tags for anything going through the QB's, it has to match specifically to the access pattern, anything that isn't in the metrics table for this query builder should fallback. `spans` and `transactions` should have their own separate set of tags.

Reply:
Yeah agreed, its why i'm just turning this off entirely with `spans` for now here
https://github.com/getsentry/sentry/pull/63789/files#diff-4cf33582e331d118cce575a23d98881984f20b579edcba5122bbcb2a831f0d28R35-R38

fyi, we have a `transaction.light` or something somewhere too that this is technically incorrect for, but we would've returning valid queries before for those anyways.